website/static/js/search.js

```diff
@@ -76,6 +76,7 @@ var ViewModel = function(params) {
     self.resultsPerPage = ko.observable(10);
     self.categories = ko.observableArray([]);
     self.searchStarted = ko.observable(false);
+    self.jsonData = ko.observable('');
```

Comment:
Need data.count and jsonData to compare. Otherwise, you can't tell if it's user hit enter too quickly or is searching for 2 different thing in very short period.